dae/dae/annotation/tests/test_annotate_columns_and_vcf.py

```diff
@@ -78,6 +78,9 @@ def annotate_directory_fixture(tmp_path):
         "annotation.yaml": """
```

Comment:
transition to `setup_directories` function https://github.com/iossifovlab/gpf/blob/7ff8254b17f8dfc33a9a41f8c45f0ea58bb8c7d5/dae/dae/genomic_resources/testing.py#L56

and remove `setup_dir` function